tasks/aws-elasticache.yml

```diff
@@ -1,18 +1,8 @@
 ---
 - name: Ensure dev packages are installed.
   apt:
-    name: "{{ item }}"
+    name: ['zlib1g-dev', 'pkg-config', 'libevent-dev', 'gcc', 'g++', 'make', 'autoconf', 'libsasl2-dev', 'git']
```

Comment:
same here

Reply:
yes, this one:

[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via
squash_actions is deprecated. Instead of using a loop to supply multiple items
and specifying name: "{{item}}", please use name: ['package', 'package'] and remove the loop.